### PR TITLE
feature/908 - add errors handling

### PR DESCRIPTION
**Pull request by AnnaEmson**
### What

[Feature 908: Users and teams screens not displaying errors when getting 5xx ](https://trello.com/c/mqiR1Dmh/908-908-users-and-teams-screens-not-displaying-errors-when-getting-5xx-errors-from-identity-api)

Screenshot 2022-03-24 at 15 07 50
Screenshot 2022-03-24 at 14 37 36

### How to review
code

### Who can review

Describe who worked on the changes, so that other people can review.

### CODE REVIEW

File: src/app/config/groups/thunks.js [hunk 1]
```diff
@@ -82,9 +83,22 @@ export const fetchGroupsRequest = isNewSignIn => dispatch => {
               })
               .catch(error => {
                   dispatch(actions.loadGroupsFailure());
-                  //TODO: map responses to user friendly by content designer
-                  if (error) {
-                      notifications.add({ type: "warning", message: error?.message || error.status, autoDismiss: 5000 });
+                  if (error.status != null && error.status === 400) {
```
Comment (gilesburdett) [inline src/app/config/groups/thunks.js:86]: Is the `!= null` instead of `!== null` deliberate?

File: src/app/config/groups/thunks.js [hunk 2]
```diff
@@ -82,9 +83,22 @@ export const fetchGroupsRequest = isNewSignIn => dispatch => {
...
+                          autoDismiss: 15000,
+                          message: errCodes.GET_GROUPS_NOT_FOUND, // TODO
+                      };
+                      notifications.add(notification);
+                  } else {
+                      const notification = {
+                          type: "warning",
+                          isDismissable: true,
+                          autoDismiss: 15000,
+                          message: errCodes.GET_GROUPS_UNEXPECTED_ERROR_SHORT, // TODO
```
Comment (gilesburdett) [inline src/app/config/groups/thunks.js:99]: These could be re-used

```
const notification = {
  type: "warning",
  isDismissable: true,
  autoDismiss: 15000,
}

if (status test) {
  notifications.add({
     ...notification,
     message: errCodes.GET_GROUPS_NOT_FOUND
  })
} else {
  notifications.add({
     ...notification,
     message: errCodes.GET_GROUPS_UNEXPECTED_ERROR_SHORT
  })
}
```

or...

```
notifications.add({
  type: "warning",
  isDismissable: true,
  autoDismiss: 15000,
  message: error.status === 400
     ? errCodes.GET_GROUPS_NOT_FOUND
     : errCodes.GET_GROUPS_UNEXPECTED_ERROR_SHORT
})
```

**Review by gilesburdett — COMMENTED** [review]
The status check could be worth revisiting

File: src/app/config/groups/thunks.js (hunk 1 above)
Comment (AnnaEmson) [reply src/app/config/groups/thunks.js:86]: Hi, that was used as the backend wasn't ready yet and the errors object were coming inconsistent. will  check if we still need it. The whole errors display will be revisited as not all messages and statuses are designed yet. this is just high level, first iteration. .

**Review by gilesburdett — APPROVED** [review]

**Pull request merged**